### ISSUE TITLE
Photosphere support

### ISSUE THREAD

**Issue by andree182** [SimpleMobileTools/Simple-Gallery#211]
If the photo is detected to be a photosphere, it would be nice if it would be possible to open such file e.g. in Google Street View (or Google Photos), like what it's done with videos...

Of course even nicer thing to do would be actually showing the pic as photosphere (parsing+rendering via OpenGL is actually not that hard... https://github.com/andree182/PythonPhotoSphere ).

**Comment by vanitasvitae** [comment]
Displaying the photosphere in-app would be super nice! Currently I cannot view photospheres because my phone has no google services. Filling this gap would be a huge addition :)

**Comment by SammysHP** [comment]
There seems to be an API for this, see https://developers.google.com/streetview/android/panorama

**Comment by SammysHP** [comment]
And here is an example: https://stackoverflow.com/questions/35295303/panorama-api-inflate-layout-on-top-of-panorama-view

**Comment by tibbi** [comment]
thanks, will check it out someday

**Comment by vanitasvitae** [comment]
Afaict this requires play services, doesn't it?

**Comment by SammysHP** [comment]
Yes, it will only work with Google-infected devices. On the other hand I don't know any app except Google Camera that can create 360 panoramas.

This should be easy to implement (just ask Google Play Services whether the current image is a Photosphere or not and show a button that starts the intent) and thus might be valuable as an intermediate solution.

**Comment by tibbi** [comment]
I will have to research photosphere, havent really worked with it yet. I definitely want to add a built in solution tho, not just redirect to some other app.

**Comment by vanitasvitae** [comment]
Niiiice!
